server/src/main/java/org/opensearch/gateway/remote/DefaultRandomObject.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.gateway.remote;
+
+import org.opensearch.common.Randomness;
+
+import java.util.Random;
+
+/**
+ * Utility to provide a {@link Random} static instance
+ *
+ * @opensearch.internal
+ */
+public class DefaultRandomObject {
```

Comment:
1. We can move this to commons package. 
2. Nit: Naming can be RandomLong if the intent is to have one instance of type long only here.